src/sdg/producten/views/product.py

```diff
@@ -264,6 +267,43 @@ def _get_generieke_taal_producten(self):
 
         return [nl, en]
 
+    def _get_published_taal_product_links(self):
+        doelgroep = self.product.generiek_product.doelgroep
+        if not doelgroep:
+            return None
+
+        product_language_urls = (
+            self.product.generiek_product.vertalingen.all().values_list("taal", "slug")
+        )
+        if not product_language_urls:
+            return None
+
+        dop = self.product.catalogus.lokale_overheid.organisatie.dop_slug
+        dpc = self.product.catalogus.lokale_overheid.organisatie.dpc_slug
+
+        urls = {}
+        for language, product_slug in product_language_urls:
+            if doelgroep == DoelgroepChoices.bedrijf:
+                if language == TaalChoices.nl:
+                    urls[
+                        language
+                    ] = f"https://ondernemersplein.kvk.nl/{product_slug}/gemeente/{dop}/"
+                elif language == TaalChoices.en:
+                    urls[
+                        language
+                    ] = f"https://business.gov.nl/regulation/{product_slug}/municipality/{dop}"
+            elif doelgroep == DoelgroepChoices.burger:
+                if language == TaalChoices.nl:
+                    urls[
+                        language
+                    ] = f"https://www.nederlandwereldwijd.nl/regelen-in-nederland/{product_slug}/gemeente-{dpc}"
+                elif language == TaalChoices.en:
+                    urls[
+                        language
+                    ] = f"https://www.netherlandsworldwide.nl/government-services-in-the-netherlands/{product_slug}/gemeente-{dpc}"
```

Comment:
Can you convert the f-string URLs to settings that also grab their values from an environment var?

Something like:

```
SDG_DOP_URL_TEMPLATE_NL = {get_config("SDG_DOP_URL_TEMPLATE_NL ", "https://ondernemersplein.kvk.nl/{product_slug}/gemeente/{org_slug}/"
SDG_DOP_URL_TEMPLATE_EN ...
SDG_DPC_URL_TEMPLATE_NL ...
SDG_DPC_URL_TEMPLATE_EN ...
```

You probably need to use `.replace` instead of f-strings.

This is needed to also make it work for provinces and water authorities.